mysql-compatibility.md

```diff
@@ -57,6 +57,7 @@ However, some features of MySQL are not supported. This could be because there i
 + `OPTIMIZE TABLE` syntax
 + `HANDLER` statement
 + `CREATE TABLESPACE` statement
++ "Session Tracker: Add GTIDs context to the OK packet". See [MySQL WL#6128: Session Tracker: Add GTIDs context to the OK packet](https://dev.mysql.com/worklog/task/?id=6128).
```

Comment:
Minor nit, but I think you can remove the WL link. It's about the low-level implementation, so won't make sense to most readers.

Reply:
```suggestion
+ "Session Tracker: Add GTIDs context to the OK packet"
```